### PR TITLE
Add pending packages to README.OmniOS

### PR DESCRIPTION
**Pull request by citrus-it**
... and remove lx merge status as this is tracked in bloody.

### CODE REVIEW

File: README.OmniOS [hunk 1]
```diff
@@ -1,72 +1,21 @@
...
-This matches a CentOS 6.8 image from Joyent.  To install the image:
+Packages:
+ > system/kernel
+ > system/kernel/platform
+ > system/test/ostest
+ > system/extended-system-utilities
 
-     zoneadm -z lx0 install -s /root/path/to/centos68.gz
+Archive:
+ > https://downloads.omniosce.org/pkg/ipsec-backport.p5p
```
Comment (oetiker) [inline README.OmniOS]: maybe add a note on how to install that archive ... ?

Comment (citrus-it) [inline README.OmniOS]: Done.

Comment (hadfl) [inline README.OmniOS]: this is just for us to keep track on the packages we have to publish later, we still have to decide if we provide archives and if so, we have to communicate and document it (e.g. on the webpage/release notes) i don't think that README.OmniOS is/should be the place for ppl to check for install info.

**Comment by oetiker** [comment]
I have squash merged this ...

**Pull request merged**